src/shared/components/results/Results.tsx

```diff
@@ -72,6 +77,36 @@ const Results = () => {
 
   const [params] = getParamsFromURL(search);
 
+  // Some cross references legitimately have colons in their ids eg PTHR34313:SF2
+  // The API returns "'PTHR34313' is not a valid search field" on those occassions
+  // so to assist the users, escape the colon PTHR34313\:SF2 and try a head request.
+  // If something found then suggest this with the DidYouMean suggestions.
+  const invalidSearchFieldQueryWithColon = isInvalidSearchFieldQueryWithColon(
+    params.query,
+    facetApiObject?.error?.response?.data?.messages,
+    ns
+  );
+  const escapedColonQuery = escapeInvalidSearchFieldQueryWithColon(
+    params.query
+  );
+  const escapedColonQueryUrl = useNSQuery(
+    invalidSearchFieldQueryWithColon
+      ? {
+          overrideQuery: escapedColonQuery,
+          withFacets: false,
+          size: 0,
+        }
+      : undefined
+  );
+  const escapedColonQueryData = useDataApi(
+    invalidSearchFieldQueryWithColon ? escapedColonQueryUrl : null,
+    {
+      method: 'HEAD',
+    }
+  );
+  const escapedColonQueryTotal =
```

Comment:
Need to parse this as a number, the header will be string values, so you could get "0" and lower in l.161 this would be truthy